.dockerignore

```diff
@@ -0,0 +1,14 @@
+.idea
+src
+target/classes/form-specs
```

Comment:
Doesn't the `.dockerignore` file support wildcards? `target/**` might be less of a maintenance burden.

Reply:
The problem is that we don't have to exclude all the files from `target` directory :-/ We want to keep the `target/fdp-spring-boot.jar` and `target/classes/application-production.yml`

Reply:
Luckely the [`.dockerignore` reference](https://docs.docker.com/engine/reference/builder/#dockerignore-file) provides a solution there:

`!target/fdp-spring-boot.jar`
`!target/classes/application-production.yml`

I'm not saying we _should_ use something like this, but it's more maintainable to explicitly whitelist 2 files than it is to manually blacklist 12 entries.

Reply:
Great! I didn't know that. I will fix it ;-)